build.sbt

```diff
@@ -2,7 +2,7 @@ import FreeGen2._
 
 // Library versions all in one place, for convenience and sanity.
 lazy val catsVersion          = "2.1.1"
-lazy val catsEffectVersion    = "2.1.4"
+lazy val catsEffectVersion    = "3.0-805b021"
 lazy val circeVersion         = "0.13.0"
 lazy val collCompatVersion    = "2.1.6"
 lazy val fs2Version           = "2.4.2"
```

Comment:
`3.0-9d83065` is compatible with this version of CE3

Reply:
Also I'm working on getting actual-M1 out right now.

Reply:
@mpilquist you mean fs2 version right?

Reply:
Yep